src/app/static/src/app/components/forms/DynamicFormMultiSelect.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div :class="cssClass">
+        <tree-select :multiple="true"
+                     :clearable="!formControl.required"
```

Comment:
Good point. I've decoupled them and set clearable to false because the cross icon gets in the way of the dropdown icon. There seems to be a bug with the treeselect hidden input - it gets updated to the most recent selected value rather than a comma separated list of all values - hence managing a hidden input manually, so the required attribute won't do us any good here.